src/infrastructure/gateway/RegistrationGateway.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
```

Comment:
what happens if the eslint is enabled here?

Reply:
this is because eslint is in error due to the methods not being implemented
once implemented with the call via the sdk, this comment will be deleted